### PR TITLE
add tests that proves that LP will get benefits

### PR DESCRIPTION
**Pull request by goncer**

### CODE REVIEW

**Review by CodeByEddy — APPROVED** [review]
@goncer Just giving you an approve in case you want to merge this, not sure if you need more feedback from @stanieltron

File: test/story/story.LP.test.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+    const userTokensAfter = testUser2
+      .getAsset(token1)
+      ?.amountAfter.free!.add(
+        testUser2.getAsset(MGA_ASSET_ID)?.amountAfter.free!
+      );
+    const totalPoolBefore = poolBalanceBeforeSwaps[0].add(
+      poolBalanceBeforeSwaps[1]
+    );
+    const totalPoolAfter = poolBalance[0].add(poolBalance[1]);
```
Comment (goncer) [inline test/story/story.LP.test.ts:102]: Feddback from Stano: this does not make sense.

Comment (goncer) [reply test/story/story.LP.test.ts:102]: Let's do the maths with one token:
250,000MGA - 250,000 USD
1MGA-1USD
do 10 swaps ->
TODO:Do reverse 10 swaps to get a balance pool
do 10 swaps <-
xMGA - yUSD
350,000-178,748.7522
lefSide x times right side.

Comment (goncer) [reply test/story/story.LP.test.ts:103]: That worked. 
the 5% was an appreciation token issue.

**Pull request merged**